packages/dds/tree/src/shared-tree/treeAlpha.ts

```diff
@@ -97,9 +100,12 @@ Object.freeze(identifier);
 /**
  * A utility interface for retrieving or converting node identifiers.
  *
+ * @remarks
  * This provides methods to:
  * - Retrieve long or short identifiers from nodes
```

Comment:
Nit: you'll need a line break here as well :/
```suggestion

 * - Retrieve long or short identifiers from nodes
```